### PR TITLE
feat: Toast appears when user rates chat

### LINKED ISSUE #2120: Toast message after giving response
**Describe the feature you'd like**
In the chat activity , after the user gives response ( thumbs up or thumbs down ) it just goes off bluntly and the user does not always rate the skill

**Additional context**
A toast message thanking the user for responding would be better so that it gives a better experience.

**Screenshots**
Screenshot_20190331-182346_SUSIAI

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by atm1504**
Fixes #2120 

Screenshots for the change: 
ezgif com-video-to-gif

### CODE REVIEW

File: app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/ChatViewHolder.kt [hunk 1]
```diff
@@ -130,8 +130,10 @@ class ChatViewHolder(view: View, clickListener: MessageViewHolder.ClickListener,
         realm.beginTransaction()
         if (which) {
             model?.isPositiveRated = what
+            Toast.makeText(contexts, R.string.rate_chat, Toast.LENGTH_SHORT).show()
         } else {
             model?.isNegativeRated = what
+            Toast.makeText(contexts, R.string.rate_chat, Toast.LENGTH_SHORT).show()
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/ChatViewHolder.kt]:
```js
if (i ==1)
  print('Yes')
else
  print('Yes')
```

What's the difference?

Comment (atm1504) [reply app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/ChatViewHolder.kt]: Sir, I am making it in one line. But, here look that it has been succesfully added to the database.

File: app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/ChatViewHolder.kt [hunk 2]
```diff
@@ -49,6 +50,7 @@ class ChatViewHolder(view: View, clickListener: MessageViewHolder.ClickListener,
     fun setView(model: ChatMessage?, viewType: Int, context: Context) {
         if (model != null) {
             this.model = model
+            contexts = context
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/ChatViewHolder.kt]: Why?

Comment (atm1504) [reply app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/ChatViewHolder.kt]: I tried to get the context via requireContext in setRatings, but I didn't get it. Also getContext, this failed. Hence, I came up with this id of storing the context in a global variable and using it in this function.

Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/ChatViewHolder.kt]: No need to store in global variable, just pass it as an argument to the function

Comment (atm1504) [reply app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/ChatViewHolder.kt]: > Why?

Okk sir

**Comment by atm1504** [comment]
@iamareebjamal sir, I further analysed it and found that the method I implemented is wrong, I will further look into it and make relevant changes.

**Pull request merged**